packages/ra-ui-materialui/src/form/FormTab.js

```diff
@@ -73,15 +73,20 @@ const FormTab = ({
 };
 
 FormTab.propTypes = {
+    basePath: PropTypes.string,
     className: PropTypes.string,
     contentClassName: PropTypes.string,
     children: PropTypes.node,
     intent: PropTypes.oneOf(['header', 'content']),
     hidden: PropTypes.bool,
     icon: PropTypes.element,
     label: PropTypes.string.isRequired,
+    margin: PropTypes.oneOf(['none', 'dense', 'normal']),
     path: PropTypes.string,
+    record: PropTypes.object,
+    resource: PropTypes.string,
     value: PropTypes.string,
+    variant: PropTypes.oneOf(['text', 'outlined', 'contained']),
```

Comment:
where did you get these values from? The [material-ui doc](https://material-ui.com/api/text-field/#props) says the possible values are `standard`, `outlined`, and `filled`

Reply:
You are right.